src/__test__/utils/processUpload.test.js

```diff
@@ -23,32 +23,35 @@ const validFilesList = [
       name: 'features.tsv.gz',
       path: '/WT13/features.tsv.gz',
       type: 'application/gzip',
-      valid: true,
     },
     upload: { status: UploadStatus.UPLOADING },
     errors: '',
+    compressed: true,
+    valid: true,
   },
   {
     name: 'WT13/barcodes.tsv.gz',
     bundle: {
-      name: 'features.tsv.gz',
+      name: 'barcodes.tsv.gz',
       path: '/WT13/barcodes.tsv.gz',
       type: 'application/gzip',
-      valid: true,
     },
     upload: { status: UploadStatus.UPLOADING },
```

Comment:
do you have any unit tests that test the behaviour when the user is trying to upload some invalid files?

Reply:
There are tests for what happens when a file it marked as invalid but not for the validation itself. I will try to add them by mocking `readFileToBuffer`.